src/utils/matmul_helper.h

```diff
@@ -349,8 +349,11 @@ class MMHelper {
         // W8A8
         else if constexpr (std::is_same_v<WeiT, w8a8_t>) {
             using dt = dnnl::memory::data_type;
+            dnnl::engine eng(dnnl::engine::kind::cpu, 0);
+            dnnl::stream stm(eng);
```

Comment:
why now need to create an engine and stream every time calling into the function? it may impact the performance.

Reply:
Use unified onednn engine and stream.